x11/rofi/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            davatorium rofi 1.7.3
+revision                0
+categories              x11
+maintainers             {@sainnhe gmail.com:sainnhe} \
+                        openmaintainer
+license                 MIT
+
+description             A window switcher, application launcher and dmenu replacement
+
+long_description \
+    Rofi, like dmenu, will provide the user with a textual list of options where \
+    one or more can be selected. This can either be running an application, \
+    selecting a window, or options provided by an external script.
+
+github.tarball_from     releases
+
+checksums               rmd160  3f119b143bbc9003fbb31c55efb98ce83060441a \
+                        sha256  608458775d0a8699dc31027e786aa9f3a63658ebfb98c5c807620a5889687ecc \
+                        size    935184
+
+depends_build-append    port:pkgconfig \
+                        port:autoconf \
+                        port:automake \
+                        port:bison \
+                        port:flex \
+                        port:check
+
+depends_lib             port:glib2 \
+                        port:libxkbcommon-x11 \
+                        port:xorg-xcb-util-wm \
+                        port:xorg-xcb-util-cursor \
+                        port:startup-notification \
+                        port:pango \
+                        port:cairo \
+                        port:gdk-pixbuf2
+
+depends_run             port:xorg-server
+
+patchfiles              patch-filebrowser.diff
+
+configure.cmd           autoreconf -i && ./configure
```

Comment:
MacPorts supports `autoreconf`, so you don't have to combine it into `configure.cmd`.

It would look something like this:

```
use_autoreconf          yes
autoreconf.args         -i
```

With that, you can remove that `configure.cmd` line.

Reply:
@herbygillot Thanks! I've fixed that.